VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -27,9 +29,9 @@
         void reportComment(DiscussionComment comment);
     }
 
-    @Inject
-    public DiscussionCommentsAdapter(Context context, Listener listener, DiscussionComment response) {
+    public DiscussionCommentsAdapter(Context context, Router router, Listener listener, DiscussionComment response) {
```

Comment:
> This still seems to be unaddressed.

I removed the Router reference from this class, so your suggestion to inject the Router is no longer relevant :P